clarity/src/vm/docs/mod.rs

```diff
@@ -1805,6 +1809,74 @@ The `addrs` list contains the same PoX address values passed into the PoX smart
 "
 };
 
+const GET_STACKS_BLOCK_INFO_API: SpecialAPI = SpecialAPI {
+    input_type: "StacksBlockInfoPropertyName, uint",
+    snippet: "get-stacks-block-info? ${1:prop} ${2:block-height}",
+    output_type: "(optional buff) | (optional uint)",
+    signature: "(get-stacks-block-info? prop-name block-height)",
+    description: "The `get-stacks-block-info?` function fetches data for a block of the given *Stacks* block height. The
+value and type returned are determined by the specified `StacksBlockInfoPropertyName`. If the provided `block-height` does
+not correspond to an existing block prior to the current block, the function returns `none`. The currently available property names
+are as follows:
+
+- `id-header-hash`: This property returns a `(buff 32)` value containing the _index block hash_ of a Stacks block.   This hash is globally unique, and is derived
+from the block hash and the history of accepted PoX operations.  This is also the block hash value you would pass into `(at-block)`.
+
+- `header-hash`: This property returns a `(buff 32)` value containing the header hash of a Stacks block, given a Stacks chain height.  **WARNING* this hash is
```

Comment:
This should be `block-hash`, since that's the nomenclature we've been using so far.